tests/Feature/MyParcelComApiTest.php

```diff
@@ -230,45 +231,46 @@ public function testGetRegions()
     {
         $regions = $this->api->getRegions();
 
-        $this->assertInternalType('array', $regions);
-        $this->assertCount(78, $regions);
-        array_walk($regions, function ($region) {
+        $this->assertInstanceOf(CollectionInterface::class, $regions);
+        $this->assertEquals(78, $regions->count());
+
+        foreach ($regions as $region) {
             $this->assertInstanceOf(RegionInterface::class, $region);
-        });
+        }
     }
 
     /** @test */
     public function testGetGbRegions()
     {
         $regions = $this->api->getRegions('GB');
 
-        $this->assertInternalType('array', $regions);
-        $this->assertCount(5, $regions);
-        array_walk($regions, function ($region) {
+        $this->assertInstanceOf(CollectionInterface::class, $regions);
+        $this->assertEquals(9, $regions->count());
```

Comment:
Did you adjust the number of regions in the stubs?

Reply:
Retrieval of the regions is now filtered in the request not after request. I added a new stub which is just raw data returned by the api when doing that request.